examples/squidpy_segment.py

```diff
@@ -0,0 +1,29 @@
+import squidpy as sq
+from ngff_tables_prototype.writer import write_spatial_anndata
+import numpy as np
+
+
+def write_segment_adata() -> None:
```

Comment:
Nitpick: can we call it `write_segmentation_adata()` instead? I makes a bit more sense grammar wise.